__tests__/__assets__/cap-proj/rest-client-test/requests.http

```diff
@@ -0,0 +1,12 @@
+# @fqdn = http://localhost:4004
+@fqdn = https://4d3dc3fctrial-dev-cap-proj-srv.cfapps.eu10.hana.ondemand.com
+###
+# "reset" aka redeploy from csv
+POST {{fqdn}}/beershop/reset
```

Comment:
With {{$dotenv scpServiceURL}} we could use a .env file located in the rest-client-test folder and not expose any trial subaccount that should not be exposed.

Reply:
👍 will do